tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py

```diff
@@ -33,6 +33,3 @@ def validate(self):
             "tedge_connect_c8y.out", "tedge-agent service successfully started and enabled!", contains=True
         )
 
-        self.assertGrep(
```

Comment:
Don't we need to check for the tedge-mapper?

Reply:
We should, I fixed the filename instead.